package/MDAnalysis/core/universe.py

```diff
@@ -1030,6 +1270,19 @@ def _fragdict(self):
                 fragdict[a.ix] = fraginfo(i, f)
 
         return fragdict
+    
+    def clear_fragments(self):
```

Comment:
Not sure this method is useful.  It shouldn't ever need to be called by a user (because it is to maintain internal book keeping) and it's a one liner anyway (`self._cache.pop('fragments', None)`)